cds/modules/records/serializers/vtt.py

```diff
@@ -64,39 +67,38 @@ def format(self):
     def _format_frames(record):
         """Select frames and format the start/end times."""
         thumbnail_data = []
-        frames = record["_files"][0]["frame"]
-
-        # sorts frames
-        frames.sort(key=lambda s: s["key"])
-        frames.sort(key=lambda s: len(s["key"]))
-
-        # uses the 5, 15... 95 % frames
-        used_frames = [frames[int(round(float((10 * n) + 5) *
-                       len(frames)/100))-1] for n in range(10)]
-
-        video_duration = float(record["_files"][0]["tags"]["duration"])
-        thumbnail_duration = round(float(video_duration/10), 3)
-
-        clipstart = 0
-        clipend = clipstart + thumbnail_duration
-
-        for i in range(10):
-            start = VTT.time_format(clipstart)
-            end = VTT.time_format(clipend)
-            clipstart = clipend
-            clipend += thumbnail_duration
-            file = used_frames[i]["links"]["self"]
-            info = {}
-            info['start_time'] = start
-            info['end_time'] = end
-            info['file_name'] = file
+        frames = record['_files'][0]['frame']
```

Comment:
I think making the assumption that the first file in the list is going to be the video is a bit dangerous, WDYT?